types/mjml-react/extensions/index.d.ts

```diff
@@ -4,3 +4,4 @@ export class MjmlComment extends React.Component<{ children: string }> { }
 export class MjmlConditionalComment extends React.Component<{ children: string; condition: string }> { }
 export class MjmlTrackingPixel extends React.Component<{ src: string }> { }
 export class MjmlYahooStyle extends React.Component<{ children: string }> { }
+export class MjmlHtml extends React.Component<{ tag: string; html: string }> { }
```

Comment:
This looks new. Where is this coming from?

Reply:
From the mjml-react extension package: https://github.com/wix-incubator/mjml-react#extensions

Implementation: https://github.com/wix-incubator/mjml-react/blob/master/src/extensions/mjml-html.js